x/multisig/types/genesis.go

```diff
@@ -25,6 +40,60 @@ func (m GenesisState) Validate() error {
 		return getValidateError(err)
 	}
 
+	keyIDs := make(map[exported.KeyID]bool, len(m.KeygenSessions)+len(m.Keys))
+	completedKeyIDs := make(map[exported.KeyID]bool)
+	for _, keygenSession := range m.KeygenSessions {
+		if keyIDs[keygenSession.GetKeyID()] {
+			return getValidateError(fmt.Errorf("duplicate key ID seen"))
+		}
+		keyIDs[keygenSession.GetKeyID()] = true
```

Comment:
Maybe do `toLower()` since we only allow a unique lowercase key ID?

Reply:
sure